### PR TITLE
Connect frontend with backend for login + integrate a new profile page

### PR DESCRIPTION
**Pull request by JonasEbbinghaus25**

### CODE REVIEW

**Review by maxschwinghammer — CHANGES_REQUESTED** [review]
React CI failing. Solved

File: src/main/java/de/tinf22b6/dhbwhub/controller/AuthController.java [hunk 1]
```diff
@@ -39,6 +39,11 @@ public class AuthController {
     @PostMapping("login")
     public ResponseEntity<?> login(@Valid @RequestBody LoginRequest loginRequest) {
 
+        System.out.println("Request:");
```
Comment (FurKay00) [inline src/main/java/de/tinf22b6/dhbwhub/controller/AuthController.java:42]: Pls delete

Comment (JonasEbbinghaus25) [reply src/main/java/de/tinf22b6/dhbwhub/controller/AuthController.java]: Deleted

**Review by maxschwinghammer — CHANGES_REQUESTED** [review]
Please adjust the design, you had it better before

**Review by FurKay00 — APPROVED** [review]

**Review by maxschwinghammer — APPROVED** [review]

**Pull request merged**